src/platform/baytrail/platform.c

```diff
@@ -239,6 +239,12 @@ int platform_init(struct sof *sof)
 #if defined CONFIG_CHERRYTRAIL
 	/* mask SSP 3 - 5 interrupts */
 	shim_write(SHIM_PIMRH, shim_read(SHIM_PIMRH) | 0x00000700);
+
+	/* Reset M/N SSP clock dividers */
```

Comment:
Do you have the defaults according to the datasheet? Otherwise I am going to read out the the values when boot-beep does not occur and write those back. Also, will these values be the same as Byt (in other words should I move it outside the CHT block?)

Reply:
I think the defaults you read back are good to use as they bypass the divider.

Reply:
Ok sounds good, and to clarify, these are going to be the same values for byt? Or should I test to see if they need different ones.

Reply:
Yes, should be the same as BYT too.

Reply:
@cujomalainey do I understand it correctly, that you were going to verify the values by reading them out? What was the result?

Reply:
no, the hardware reset values were re-applied